src/routes/StyleGuide.jsx

```diff
@@ -117,6 +127,69 @@ const StyleGuide = () => {
           </Paragraph>
         </Modal>
       </Flex>
+
+      <Divider />
+
+      <Flex justify="flex-start" className="px-12">
+        <Heading level={2}>Input</Heading>
+      </Flex>
+      <Flex vertical justify="flex-start" gap="middle" className="px-12">
+        <CustomInput
+          className="w-80"
+          placeholder="Basic Input"
+          allowClear
+          onChange={onChange}
+        />
+
+        <CustomInputWithLabel
+          className="w-80"
+          content="Input with label"
+          placeholder="Input with label"
+        />
+
+        <CustomInputWithPassword
+          className="w-80"
+          placeholder="input password"
+          iconRender={(visible) =>
+            visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />
+          }
+        />
+
+        <CustomInputWithSubmit
+          className="w-80"
+          placeholder="Custom input with submit button"
+        />
+      </Flex>
+      <Divider />
+      <Flex justify="flex-start" className="px-12">
+        <Heading level={2}>Checkbox</Heading>
+      </Flex>
+      <Flex vertical justify="flex-start" gap="middle" className="px-12">
+        <CustomCheckbox className="w-60" content="CheckBox Item" />
+      </Flex>
+
+      <Divider />
+
+      <Flex justify="flex-start" className="px-12">
+        <Heading level={2}>Select</Heading>
+      </Flex>
+
+      <Flex vertical justify="flex-start" gap="middle" className="px-12">
```

Comment:
Make text left align as in the style guide

Reply:
> Make text left align as in the style guide

Changed. Will reflect in the next PR, wait for 10 mins